File: SDK/php/src/Types/VerifyOptions.php
<?php

namespace Verifio\Types;

class VerifyOptions
{
    public ?bool $skipDisposable = null;
    public ?bool $skipRole = null;
    public ?bool $skipTypo = null;

    public function toArray(): array
    {
        $opts = [];
        if ($this->skipDisposable !== null) $opts['skipDisposable'] = $this->skipDisposable;
        if ($this->skipRole !== null) $opts['skipRole'] = $this->skipRole;
        if ($this->skipTypo !== null) $opts['skipTypo'] = $this->skipTypo;
        return $opts;
    }
}

File: SDK/java/src/main/java/com/reloop/verifio/types/VerifyOptions.java
package com.reloop.verifio.types;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class VerifyOptions {
    private Boolean skipDisposable;
    private Boolean skipRole;
    private Boolean skipTypo;

    public Boolean getSkipDisposable() { return skipDisposable; }
    public void setSkipDisposable(Boolean skipDisposable) { this.skipDisposable = skipDisposable; }
    public Boolean getSkipRole() { return skipRole; }
    public void setSkipRole(Boolean skipRole) { this.skipRole = skipRole; }
    public Boolean getSkipTypo() { return skipTypo; }
    public void setSkipTypo(Boolean skipTypo) { this.skipTypo = skipTypo; }
}
